src/BDD/integerset.jl

```diff
@@ -0,0 +1,224 @@
+# Manipulation of sets of integers
+
+## IntSet
+"""
+    mutable struct IntSet <: AbstractSet{<:Integer}
+
+Same as `Base.Set{<:Integer}` but with `CUDD`.
+"""
+mutable struct IntSet{T<:Integer} <: AbstractSet{T}
```

Comment:
Let's try to minimize the amount of renaming and make smaller PRs :)
Julia v0.6's `IntSet` was renamed to `BitSet`, the naming `BDD.BitSet` was consistent with this even if it is not with `BDD.IntTupleSet`.
When you want to rename, try to open an issue first to justify the choice and then if there is a good reason, we make a PR that only does the renaming.
Another point to consider is that here the files are marked as completely different in the diff. You should do `git mv BitSet.jl integerset.jl` so that `git` understands that they are the same files and computes diff between the two even if the names were changed.